development/networkclient.py

```diff
@@ -246,7 +246,7 @@ def onhelp(*args):
 except (ValueError, IndexError):
   port = 0
 
-if host == None or port == None or port <= 0:
+if None in [host, port] or port <= 0:
```

Comment:
I don't like that one to be honest. The previous code should be improved as well, by using `is` for comparison. It should be

```python
if host is None or port is None
```